core/kazoo_web/src/kz_http.erl

```diff
@@ -279,7 +279,11 @@ build_request(Method, Url, Headers, Body) when (Method == 'post');
                                           ]
                                          ,Headers
                                          ,""),
-    {kz_util:to_list(Url), ensure_string_headers(Headers), kz_util:to_list(ContentType), Body}.
+    {kz_util:to_list(Url)
+    ,ensure_string_headers(Headers)
+    ,kz_util:to_list(ContentType)
+    ,kz_util:to_binary(Body)
```

Comment:
How about using `iolist_to_binary` instead?
Also, how about fixing all the calls to kz_http:req calls that call to_binary(Body)? This should save some time/confusion.